src/client/src/App/Analytics/ProfitAndLoss/__tests__/ProfitAndLoss.test.tsx

```diff
@@ -50,37 +48,37 @@ describe("Profit and Loss", () => {
     analyticsMock.__resetMocks()
   })
 
-  it("renders the chart correctly", () => {
+  it.skip("renders the chart correctly", () => {
```

Comment:
I think this test should removed.  It was relying on `react-test-renderer`, and it doesn't seem to be testing anything meaningful (look at the snapshot that is being deleted).

Reply:
same for Positions test .. just had to skip for unit tests on Windows, as I see you have in this PR .. precise position values are not consistent across o/s .. who knows why? pointless testing these unless we have logic that depends on the exact position